pkg/controller/compliancescan/compliancescan_controller_test.go

```diff
@@ -501,9 +501,11 @@ var _ = Describe("Testing compliancescan controller phases", func() {
 
 				// Set state to DONE
 				compliancescaninstance.Status.Phase = compv1alpha1.PhaseDone
-				compliancescaninstance.Spec.Debug = true
 				err := reconciler.Client.Status().Update(context.TODO(), compliancescaninstance)
 				Expect(err).To(BeNil())
+				compliancescaninstance.Spec.Debug = true
+				err = reconciler.Client.Update(context.TODO(), compliancescaninstance)
```

Comment:
Splitting this into two updates was necessary because of recent changes to fake client:

https://github.com/kubernetes-sigs/controller-runtime/pull/2259